example/custom-delims/composition.yaml

```
@@ -0,0 +1,68 @@
+apiVersion: apiextensions.crossplane.io/v1
+kind: Composition
+metadata:
+  name: example-inline
+spec:
+  compositeTypeRef:
+    apiVersion: example.crossplane.io/v1beta1
+    kind: XR
+  mode: Pipeline
+  pipeline:
+    - step: render-templates
+      functionRef:
+        name: function-go-templating
+      input:
+        apiVersion: gotemplating.fn.crossplane.io/v1beta1
+        kind: GoTemplate
+        leftDelims: '[['
+        rightDelims: ']]'
```

```suggestion
      delims:
        left: '[['
        right: ']]'
```

or should we even nest under `config` or something if we envision more configuration in the future?

```
config:
  delims:
    left: '[['
    right: ']]'
```